### PR TITLE
Added HTTPS, http is no longer supported

### PR DESCRIPTION
**Pull request by AmiT177**

### CODE REVIEW

File: steamauth/steamauth.php [hunk 1]
```diff
@@ -21,14 +21,14 @@ function loginbutton($buttonstyle = "square") {
...
-			$openid->identity = 'http://steamcommunity.com/openid';
+			$openid->identity = 'https://steamcommunity.com/openid';
 			header('Location: ' . $openid->authUrl());
 		} elseif ($openid->mode == 'cancel') {
 			echo 'User has canceled authentication!';
 		} else {
 			if($openid->validate()) { 
 				$id = $openid->identity;
-				$ptn = "/^http:\/\/steamcommunity\.com\/openid\/id\/(7[0-9]{15,25}+)$/";
+				$ptn = "/^https:\/\/steamcommunity\.com\/openid\/id\/(7[0-9]{15,25}+)$/";
```
Comment (Marcuzz) [inline steamauth/steamauth.php]: This should use https? instead of https incase they decide to switch again

**Comment by RedSparr0w** [comment]
userinfo.php line 4 will also need to be updated

**Comment by SmItH197** [comment]
Thanks for this, I'll take a look now!

**Comment by SmItH197** [comment]
Hi, I've merged #218 instead as it made more changes and included the above suggestions from @Marcuzz and @RedSparr0w

**Pull request closed without merging**